### ISSUE TITLE
PyTorch-Optimized: Cross-Entropy Loss

### ISSUE THREAD

**Issue by gbiagini** [lgragert/nn-sero-pytorch#12]

**Comment by gbiagini** [comment]
Currently using BCEWithLogits loss function (binary cross entropy) for multi-label multiclass classification.
